data/core/src/commonMain/kotlin/com/tunjid/heron/data/repository/TimelineRepository.kt

```diff
@@ -566,6 +570,10 @@ internal class OfflineTimelineRepository(
                                 val postUris = postThread.map {
                                     it.entity.uri
                                 }.toSet()
+                                fetchMissingEmbeddedRecords(
```

Comment:
This part of this method reads from the database, it shouldn't simultaneously write. The embedded records should already exist when fetching.

This method should be removed. Instead, in `SavePostView.kt`, make sure any embedded records are also added to the `MultipleEntitySaver`.